relay/nakama/main.go

```diff
@@ -341,9 +349,10 @@ func handleClaimPersona(ptv *personaTagVerifier, notifier *receiptNotifier) naka
 func handleShowPersona(ctx context.Context, logger runtime.Logger, _ *sql.DB, nk runtime.NakamaModule, _ string,
 ) (string, error) {
 	ptr, err := loadPersonaTagStorageObj(ctx, nk)
-	if eris.Is(eris.Cause(err), ErrPersonaTagStorageObjNotFound) {
-		return logErrorMessageFailedPrecondition(logger, err, "no persona tag found")
-	} else if err != nil {
+	if err != nil {
+		if eris.Is(eris.Cause(err), ErrPersonaTagStorageObjNotFound) {
+			return logErrorMessageFailedPrecondition(logger, err, "no persona tag found")
+		}
```

Comment:
seems redundant. if err == nil, the second statement under it will return false.